src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj

```diff
@@ -9,12 +9,12 @@
     <PackProject>true</PackProject>
     <IncludeInVsix>true</IncludeInVsix>
     <authors>Microsoft</authors>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</PackageLicenseUrl>
-    <PackageProjectUrl	>https://github.com/NuGet/NuGet.Client</PackageProjectUrl	>
+    <PackageLicenseUrl>https://aka.ms/nugetlicense</PackageLicenseUrl>
+    <PackageProjectUrl>https://aka.ms/nugetprj</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <AssemblyDescription>APIs for invoking NuGet Restore Manager in Visual Studio.</AssemblyDescription>
-    <Description>APIs for invoking NuGet Restore Manager in Visual Studio.</Description>
-    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <AssemblyDescription>APIs for invoking NuGet Restore Manager in Visual Studio. Microsoft holds the copyright for this NuGet package. .NET Foundation holds the copyright to the source.</AssemblyDescription>
```

Comment:
I don't get it. 
Why should the description contain copyright information? 
What's the copyright field for then?

Reply:
@karann-msft  https://github.com/NuGet/Home/issues/6518#issuecomment-408205989